### PR TITLE
Updated wiki with missing references

### PR DESCRIPTION
**Pull request by MARTINI1**
## Description

Add links for online tools in geofence section, new commands pokemon-spinning and account-max-spins added in tutorial and commandline sections and add information of wh filter types and example of filtering pokemon in webhook section.

## Motivation and Context

I searched for some information on the wiki and realized that it was missing things.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/extras/Webhooks.md [hunk 1]
```diff
@@ -28,6 +28,47 @@ Add `-wh http://my-webhook/location` argument when starting RocketMap (runserver
...
+* `lure`
+
+Example for send to wh the info of pokemon, gym, raid and gym-info:
+
+```
+wh-types:[pokemon, gym, raid, gym-info]
+```
+
+## Filtering Pokemon
+To not send all the pokemon found by the RocketMap to our wh we can make use of a black/white list depending on if we want to block/allow the sending of a certain pokemon.
```
Comment (sebastienvercammen) [inline docs/extras/Webhooks.md]: > To filter what Pokémon get sent to the webhooks, you can configure a whitelist of Pokémon IDs (one per line).
>
> For example, if we only want to sent data on dratini, dragonair, dragonite, larvitar, pupitar and tyranitar, add this to your configuration:
> 
> ```
> webhook-whitelist-file: pokemon-whitelist.txt
> ```
> 
> And add the Pokémon IDs to the `pokemon-whitelist.txt` file:
> 
> ```
> 147
> 148
> 149
> 246
> 247
> 248
> ```

File: docs/first-run/tutorial.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
 # Tutorial Completion
 ## RocketMap's tutorial completion for new accounts
 RocketMap now completes the tutorial steps on all accounts on first log in, there is no more `-tut` argument to complete the tutorial.
+
 It's recommended to enable pokestop spinning in the config to get your accounts to level 2.
+
+For this, use -spin in commandline or set in the config file:
```
Comment (sebastienvercammen) [inline docs/first-run/tutorial.md]: > To enable Pokéstop spinning, add `pokestop-spinning` to your configuration file, or `-spin` to your cli parameters.

File: docs/first-run/tutorial.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+```
+pokestop-spinning: true
+```
```
Comment (sebastienvercammen) [inline docs/first-run/tutorial.md]: Remove `true`, just comment or uncomment the line to enable/disable:
```
pokestop-spinning
```

File: docs/first-run/tutorial.md [hunk 4]
```diff
@@ -1,4 +1,17 @@
...
+
+You can also set the maximum number of Pokestop spins per hour, which by default is 80, and for example, lower it to 30, putting in the commandline -ams 30 or in the config file:
```
Comment (sebastienvercammen) [inline docs/first-run/tutorial.md]: > To set the maximum number of Pokéstop spins per account per hour (default: 80), add `-ams 30` to your cli parameters or edit your configuration file:

```
account-max-spins: 30
```

**Comment by MARTINI1** [comment]
thanks for your help Seb, all done

**Pull request merged**